subsys/net/ip/tcp.c

```diff
@@ -1925,6 +1925,10 @@ static int send_reset(struct net_context *context,
 
 /* This is called when we receive data after the connection has been
  * established. The core TCP logic is located here.
+ *
+ * Prototype:
+ * enum net_verdict FUNC(struct net_conn *conn, struct net_pkt *pkt,
```

Comment:
Although this is quite pointless patch, it would be better to place the real function name here instead of FUNC so
`enum net_verdict tcp_established(struct net_conn *conn, struct net_pkt *pkt, void *user_data)`

Reply:
> Although this is quite pointless patch

Well, I found it to be a source of regular confusion for me. Not too big, but definitely somewhat annoying. As we're set on cleanup and improvement of the IP stack, this is one of my contributions to that cause.

> better to place the real function name here instead of FUNC